.codesandbox/ci.json

```diff
@@ -1,7 +1,7 @@
 {
   "buildCommand": "build:codesandbox",
   "installCommand": "install:codesandbox",
-  "node": "12",
+  "node": "16",
```

Comment:
It could be good to have a standalone PR like in https://github.com/mui/mui-x/pull/4999